store/tikv/coprocessor.go

```diff
@@ -93,16 +93,36 @@ func (c *CopClient) Send(ctx context.Context, req *kv.Request, vars *kv.Variable
 		it.respChan = make(chan *copResponse, it.concurrency)
 		it.sendRate = newRateLimit(it.concurrency)
 	}
+	workersCond := sync.NewCond(&sync.Mutex{})
+	it.actionOnExceed = &taskRateLimitAction{
+		cond: struct {
+			*sync.Cond
+			once             sync.Once
+			exceed           bool
+			tearedTicket     uint
```

Comment:
We can define this as `tokenNum` which indicates valid token num, and init it as it.concurrency, it may be more readable.

teared --> isTokenBurned

Reply:
updated.